receiver/awscontainerinsightreceiver/factory.go

```diff
@@ -42,6 +42,9 @@ const (
 
 	// Don't enable EKS control plane metrics by default
 	defaultEnableControlPlaneMetrics = false
+
+	// Don't enable Neuron metrics by default
+	defaultEnableAwsNeuronMetrics = false
```

Comment:
unlike nvidia, neuron is turned off out of the box unless customers manually turn it on?

Reply:
I think yes , it should be same a as GPU

Reply:
The default value here in contrib is false, but the agent will set it to true by default if Enhanced Container Insights is enabled.